src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filteredTodos: Todo[];
```

Comment:
This component may not know that todos are filtered. The only purpose of this component is to render some todos. I'd recommend to name this prop just "todos"